atlas_of_innovation/settings/base.py

```diff
@@ -68,10 +68,10 @@
     'default': {
```

Comment:
Can this be separated by environments? It may be that the original values of the project are the ones that Lyla uses in production (I hope not!). But we should be able to set different values for different environments (basically production, development, test, but could be more).